tidb-cluster-on-centos/azuredeploy.json

```diff
@@ -0,0 +1,467 @@
+      }
+    },
+    "tikvCount": {
+      "type": "int",
+      "minValue": 1,
+      "maxValue": 20,
+      "metadata": {
+        "description": "The numble of the tikv instances to be deployed on the cluster"
+      }
+    },
+    "virtualNetworkName": {
+      "type": "string",
+      "defaultValue": "tidbVirtNet",
+      "metadata": {
+        "description": "The arbitrary name of the virtual network provisioned for the Redis cluster"
+      }
+    },
+    "addressPrefix": {
+      "type": "string",
+      "defaultValue": "10.0.0.0/8",
+      "metadata": {
+        "description": "The network address space for the virtual network"
+      }
+    },
+    "routeTableName": {
+      "type": "string",
+      "defaultValue": "routetable",
+      "metadata": {
+        "description": "The route table for the virtual network"
+      }
+    },
+    "pdSubnetName": {
+      "type": "string",
+      "defaultValue": "pdSubnet",
+      "metadata": {
+        "description": "Subnet name for the pd virtual network that resources will be provisioned in to"
+      }
+    },
+    "pdSubnetPrefix": {
+      "type": "string",
+      "defaultValue": "10.0.240.0/24",
+      "metadata": {
+        "description": "Address space for the pd virtual network subnet"
+      }
+    },
+    "pdVmSize": {
+      "type": "string",
+      "defaultValue": "Standard_A4_v2",
+      "metadata": {
+        "description": "Virtual machine size for the pd virtual machine"
+      }
+    },
+    "monitorVmSize": {
+      "type": "string",
+      "defaultValue": "Standard_A3",
+      "metadata": {
+        "description": "Virtual machine size for the monitor virtual machine"
+      }
+    },
+    "tidbSubnetName": {
+      "type": "string",
+      "defaultValue": "tidbSubnet",
+      "metadata": {
+        "description": "Subnet name for the tidb virtual network that resources will be provisioned in to"
+      }
+    },
+    "tidbSubnetPrefix": {
+      "type": "string",
+      "defaultValue": "10.240.0.0/16",
+      "metadata": {
+        "description": "Address space for the tidb virtual network subnet"
+      }
+    },
+    "tidbVmSize": {
+      "type": "string",
+      "defaultValue": "Standard_F4s",
+      "metadata": {
+        "description": "Virtual machine size for the tidb virtual machine"
+      }
+    },
+    "tikvSubnetName": {
+      "type": "string",
+      "defaultValue": "tikvSubnet",
+      "metadata": {
+        "description": "Subnet name for the tikv virtual network that resources will be provisioned in to"
+      }
+    },
+    "tikvSubnetPrefix": {
+      "type": "string",
+      "defaultValue": "10.248.0.0/16",
+      "metadata": {
+        "description": "Address space for the tikv virtual network subnet"
+      }
+    },
+    "tikvVmSize": {
+      "type": "string",
+      "defaultValue": "Standard_D12_v2",
+      "metadata": {
+        "description": "Virtual machine size for the tikv virtual machine"
+      }
+    },
+    "artifactsLocation": {
```

Comment:
See [BP guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#samples-that-contain-extra-artifacts-custom-scripts-nested-templates-etc) for how to handle artifacts